src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -0,0 +1,64 @@
+- @pagetitle = 'Show Token'
+
+.card
+  .card-body
+    %h3.mb-3= @pagetitle
+
+    .row
+      .col-12.col-md-10.col-lg-8
+        %p.font-weight-bold Your new OBS Personal Access Token
+
+        .form-row
+          .col-sm
+            %label.font-weight-bold
+              Id:
+            %span
+              = @token.id
+
+        - if session[:show_token]
+          - session[:show_token] = nil # Ensure we're only showing the token once
+          .form-row
+            .col-12.col-md-10.col-lg-9
+              %fieldset.form-group
+                %label.col-form-label.font-weight-bold
+                  Secret:
+                .input-group
+                  = render CopyToClipboardInputComponent.new(token_string: @token.string)
+              %hr
+
+        .form-row
+          .col
+            .form-group
+              %label.col-form-label.mr-2.font-weight-bold
+                Type:
+              %span
+                = @token.type
```

Comment:
We should be consistent with the other views when displaying the token type. On the `new` view, we do it this way:

```suggestion
                = @token.type.constantize.token_name
```